src/Kaleidoscope-LEDEffect-FunctionalColor.cpp

```diff
@@ -356,8 +356,8 @@ void FunctionalColor::brightnessDown(uint8_t keyState) {
 }
 
 //refreshAt accepts a Key position and sets it to the appropriate color from a lookup function
-void FunctionalColor::refreshAt(byte r, byte c) {
-  Key k = Layer.lookupOnActiveLayer(r, c);
+void FunctionalColor::refreshAt(KeyAddr key_addr) {
```

Comment:
The way keys are addressed has changed recently. I fixed this here to suppress warnings.